proctor-pipet/proctor-pipet-core/src/main/java/com/indeed/proctor/pipet/core/var/Converter.java

```diff
@@ -25,12 +26,12 @@ public Converter(final List<ContextVariable> contextList) {
         this.contextList = contextList;
     }
 
-    public ConvertedParameters convert(final RawParameters raw) {
```

Comment:
As I mentioned in AbstractProctorLoader.java, I suggest keeping the signatures of the methods in this class and instead pass empty set to `ProctorConsumerUtils.parseForceGroupsList`.